proxy/src/main/java/com/velocitypowered/proxy/config/VelocityConfiguration.java

```diff
@@ -274,11 +274,7 @@ public boolean shouldQueryShowPlugins() {
   @Override
   public net.kyori.adventure.text.Component getMotd() {
     if (motdAsComponent == null) {
-      if (motd.startsWith("{")) {
-        motdAsComponent = GsonComponentSerializer.gson().deserialize(motd);
-      } else {
-        motdAsComponent = MiniMessage.miniMessage().deserialize(motd);
-      }
+      motdAsComponent = MiniMessage.miniMessage().deserialize(motd);
```

Comment:
There's a deserializeOrNull method you can use just to save the if check here

Reply:
I think it is better to leave it as it is, since the only time the motd of the configuration changes, is when the whole configuration is reloaded, so it would be an unnecessary waste of resources to always call the `MiniMessage#deserialize` or `#deserializeOrNull` method if the motd will always be the same